src/Aquifer.API/Endpoints/Marketing/Unsubscribe/Endpoint.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Aquifer.Data;
+using FastEndpoints;
+
+namespace Aquifer.API.Endpoints.Marketing.Unsubscribe;
+
+public class Endpoint(AquiferDbContext dbContext, ILogger<Endpoint> logger) : Endpoint<Request>
+{
+    public override void Configure()
+    {
+        Get("/marketing/unsubscribe/{UnsubscribeId}");
+        AllowAnonymous();
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        var subscriberToDisable = dbContext.ContentSubscribers.SingleOrDefault(s => s.UnsubscribeId == req.UnsubscribeId);
+        if (subscriberToDisable is not null)
+        {
+            subscriberToDisable.Enabled = false;
+            await dbContext.SaveChangesAsync(ct);
+        }
+        await SendOkAsync("You have been successfully unsubscribed.", ct);
```

Comment:
I disagree about removing that try/catch.

The thing that could potentially blow up here is the save, in which case someone legitimately trying to unsubscribe wouldn't be, and we could then take some manual action for that. If the SingleOrDefault doesn't find anything then it'll just move on.

Reply:
And if it does blow up, we can send back some nice message to try again or whatever, whereas now if it blows up they just get a 500. Remember that this is something the user is going to directly see in their browser.